lib/jormungandr/test/data/jormungandr/stake_pools/create-stake-pool.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash

set -euo pipefail
```

This will make scripts easier to debug.

If someone runs this without an output directory argument, it is going to create the keys in `~` - slightly confusing.